core/state/statedb.go

```diff
@@ -1477,3 +1481,18 @@ func (s *StateDB) PointCache() *utils.PointCache {
 func (s *StateDB) Witness() *stateless.Witness {
 	return s.witness
 }
+
+func (s *StateDB) NewAccessEvents() *AccessEvents {
+	return &AccessEvents{
+		branches:   make(map[branchAccessKey]mode),
+		chunks:     make(map[chunkAccessKey]mode),
+		pointCache: utils.NewPointCache(100),
+	}
+}
```

Comment:
make the constructor part of the package (which it is on kaustinen-with-shapella) and don't export it here because it's only used in `New`

Reply:
that won't work, because it's used in`evm.go` in order to initialize the "tx local" access witness.